addons/payment/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,101 @@
+---Fields in module 'payment'---
+payment      / payment.acquirer         / auto_confirm (selection)      : NEW required: required, selection_keys: ['at_pay_confirm', 'at_pay_now', 'none'], req_default: at_pay_confirm
+# Nothing to do
+
+payment      / payment.acquirer         / cancel_msg (html)             : NEW
+# Nothing to do
+
+payment      / payment.acquirer         / done_msg (html)               : NEW
+# Nothing to do
+
+payment      / payment.acquirer         / error_msg (html)              : NEW
+# Nothing to do
+
+payment      / payment.acquirer         / image (binary)                : NEW
+# Nothing to do
+
+payment      / payment.acquirer         / pending_msg (html)            : NEW
+# Nothing to do
+
+payment      / payment.acquirer         / registration_view_template_id (many2one): NEW relation: ir.ui.view
+# Nothing to do
+
+payment      / payment.acquirer         / sequence (integer)            : NEW
+# Nothing to do
+
+payment      / payment.acquirer         / validation (selection)        : DEL selection_keys: ['automatic', 'manual']
+# Column renamed -> done
+
+payment      / payment.method           / acquirer_id (many2one)        : NEW relation: payment.acquirer, required: required
+# Nothing to do
+
+payment      / payment.method           / acquirer_ref (char)           : NEW required: required
+# Nothing to do
+
+payment      / payment.method           / active (boolean)              : NEW
+# Nothing to do
+
+payment      / payment.method           / name (char)                   : NEW
+# Nothing to do
+
+payment      / payment.method           / partner_id (many2one)         : NEW relation: res.partner, required: required
+# Nothing to do
+
+payment      / payment.method           / payment_ids (one2many)        : NEW relation: payment.transaction
+# Nothing to do
```

Comment:
What is the inverse of this one2many? Was it already present in v8? If not, we should try to fill it heuristically. If yes, please add here apart from `Nothing to do` an explanation like `Inverse field <x> was already present on v8`